eslint.config.mjs

```diff
@@ -0,0 +1,177 @@
+import js from "@eslint/js";
+import json from "@eslint/json";
+import eslintConfigPrettier from "eslint-config-prettier";
+import perfectionist from "eslint-plugin-perfectionist";
+import react from "eslint-plugin-react";
+import globals from "globals";
+import tseslint from "typescript-eslint";
+
+export default tseslint.config(
+  {
+    name: "nl-design-system/global-ignores",
+    ignores: ["**/dist/", "**/build/", "**/coverage/"],
+  },
+  {
+    name: "rotterdam/global-ignores",
+    ignores: [
+      "**/build/",
```

Comment:
Is het nodig om `build`, `coverage` en `dist` dubbel te doen hier in combinatie met `nl-design-system/global-ignores`?